docker/docker-compose.yml

```diff
@@ -166,3 +166,26 @@ services:
       - 6832:6832/udp # accept jaeger.thrift over binary thrift protocol
       - 5778:5778 # serve configs
       - 16686:16686 # serve frontend
+
+  loki:
+    image: grafana/loki:latest
+    profiles:
+      - metrics
+    container_name: loki
+    restart: unless-stopped
+    ports:
+      - 3100:3100
+    volumes:
+      - ./assets/loki/:/etc/loki/
+    command: -config.file=/etc/loki/config.yml
+
+  promtail:
+    image: grafana/promtail:latest
+    profiles:
+      - metrics
+    container_name: promtail
+    volumes:
+      - ./assets/promtail/:/etc/promtail/
+      - ./data/promtail/log:/var/log
+    command: -config.file=/etc/promtail/config.yml
+    
```

Comment:
Check formatter?

Reply:
> When I try to run Chronicle with Loki on my VPS and test the Loki data source, I get the following error:
> 
> > Data source connected, but no labels received. Verify that Loki and Promtail is configured properly.
> 
> Is there something that I still have to set up?

No, it just says that if there was no data received yet.